src/harshalRane/Tests/FindNumbers.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Test - 2: 4th Sep'2022
+void findRequiredNumbers(int targetNum){
+	int count = 0;
+	while(sum<targetNum){		
+		count++;
+	}
+	sop(count);
+}
+----------------------------
+findRequiredNumbers(50);
+
+input : 50
+output : 10
+
+input : 17
+output : 6
+
+1+2+3+4+5+6+7+8+9+10 = 55
+ */
+package harshalRane.Tests;
+
+class FindNumbers{
+
+	void findRequiredNumbers(int targetNum){
+		int count=0;
+		int sum =0;
+		int num = 1;
+		
+		while(sum<targetNum)
+		{
+			sum = sum + num;
+			num++;
+			if(sum!=targetNum)
```

Comment:
Looks like this condition not require, please let me know your understanding Skype

Reply:
Yes Corrected my code
![image](https://user-images.githubusercontent.com/112226057/188344629-c6e30e88-b71a-4b89-ad50-9ef46709f45e.png)